src/migrations/20191112101055-create-user.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
```

Comment:
Mark required fields as `allowNull: false`. This enforces validation in db level.